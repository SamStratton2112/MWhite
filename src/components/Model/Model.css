.imgContainer{
    background-color: rgb(13, 12, 12, .9);
    height: 35em;
}

