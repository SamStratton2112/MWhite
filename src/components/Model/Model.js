import { Container } from "react-bootstrap";
import LeftImg from "./LeftImg";
import RightImg from "./RightImg";
import mike from "../../data";

const Model = ()=>{
    return(
        <Container>
            <h1 style={{textAlign:'left'}}>Modeling Experience</h1>
            <LeftImg list={mike.modeling.publications.list} image={mike.modeling.publications.image} name={mike.modeling.publications.name}/>
            <RightImg list={mike.modeling.shows.list} image={mike.modeling.shows.image} name={mike.modeling.shows.name}/>
            <LeftImg list={mike.modeling.host.list} image={mike.modeling.host.image} name={mike.modeling.host.name}/>
            <RightImg list={mike.modeling.promo.list} image={mike.modeling.promo.image} name={mike.modeling.promo.name}/>
        </Container>
    )
};

export default Model;