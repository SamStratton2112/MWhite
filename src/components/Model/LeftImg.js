import { Col, Row } from "react-bootstrap";
import List from "../List";
import './Model.css'

/** Used alongside RightImg to create a tiling effect alternating image and list.  */

const LeftImg = ({ image, list, name }) => {
    return (
        <Row className="center" >
            <Col className="center imgContainer">
                <img style={{ height: '30em' }} src={image} alt='Mike White Model' />
            </Col>
            <Col className="txtContainer">
                    <h2>{name}</h2>
                    <List list={list}/>
            </Col>
        </Row>
    )
};

export default LeftImg;