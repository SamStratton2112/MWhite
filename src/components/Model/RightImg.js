import { Row, Col } from "react-bootstrap";
import List from "../List";

const RightImg =({image, list, name})=>{
    return(
        <Row className="center">
            <Col>
                <h2>{name}</h2>
                <List list={list}/>
            </Col>
            <Col className="center imgContainer">
                <img style={{height:'30em'}} src={image} alt='Mike White Model'/>
            </Col>
        </Row>
    )
};

export default RightImg;