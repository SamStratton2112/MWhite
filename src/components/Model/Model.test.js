import Model from './Model';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

describe(Model, () => {
    it("renders without crashing", () => {
        render(<MemoryRouter>
            <Model />
        </MemoryRouter>);
    });
    it("matches snapshot", () => {
        const { asFragment } = render(<MemoryRouter>
            <Model />
        </MemoryRouter>);
        expect(asFragment()).toMatchSnapshot();
    });
});