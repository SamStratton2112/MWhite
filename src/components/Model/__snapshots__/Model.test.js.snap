// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="center row"
    >
      <div
        class="center imgContainer col"
      >
        <img
          alt="Mike White Model"
          src="https://i.imgur.com/gWLkqYS.jpeg"
          style="height: 30em;"
        />
      </div>
      <div
        class="txtContainer col"
      >
        <h2>
          Fashion Shows:
        </h2>
        <ul
          class="leftText scrollText"
          style="border-radius: .5em; color: black;"
        >
          <li>
            Scope Magazine's 6th Anniversary Party, @ Club Utopia - May 1998 (Assistant Choreographer, Model Assistant, & Runway Model)
          </li>
          <li>
            Buffalo Exchange Vintage Couture, @ Mariposa Cafe's Grand Opening - 1997
          </li>
          <li>
            Buffalo Exchange Vintage Couture @ Club Utopia - 1996
          </li>
          <li>
            S.I.A. Ski Convention, Janet Racy N.Y.C. (Runway Model) - 1995
          </li>
          <li>
            Buffalo Exchange Vintage Couture, @ Cafe' Espresso Roma - 1995
          </li>
        </ul>
      </div>
    </div>
    <div
      class="center row"
    >
      <div
        class="col"
      >
        <h2>
          Publications: 
        </h2>
        <ul
          class="leftText scrollText"
          style="border-radius: .5em; color: black;"
        >
          <li>
            RNT Architects Magazine (for SDGE) - March 2017
          </li>
          <li>
            Las Vegas Weekly Vol 10, Issue 20 'These Kicks Are Made For Dancing' Promotional photoshoot for RA & BABY'S nightclub & the ATTIC clothing stores
          </li>
          <li>
            The PAPER Las Vegas' It Magazine, Y2K Turn of The Century Issue
          </li>
          <li>
            Scope Magazine's Sixth Anniversary Party - May 1998
          </li>
          <li>
            GUESS? Jeans (Photoshoot for Retail Store Poster) - 1997
          </li>
          <li>
            SKI Show Daily - Vol 16 #3 - Mar 1996
          </li>
          <li>
            'Smoke & Cigar' Magazine - The Drink Nightclub Ad - Jan 1996
          </li>
          <li>
            Los Angeles & New York Times - Photoshoot for Venus in Furs - Sept 1995
          </li>
          <li>
            Transworld Snowboard Magazine (shot in clothes used in fashion show for S.I.A.) - 1995
          </li>
          <li>
            CALVIN KLEIN Billboard CK1 Advertisement - June 1994
          </li>
          <li>
            Exotic hair designs by DIVA Hair Salon - Details & Detour magazine - May 1994
          </li>
        </ul>
      </div>
      <div
        class="center imgContainer col"
      >
        <img
          alt="Mike White Model"
          src="https://i.imgur.com/NPhe42Z.jpeg"
          style="height: 30em;"
        />
      </div>
    </div>
    <div
      class="center row"
    >
      <div
        class="center imgContainer col"
      >
        <img
          alt="Mike White Model"
          src="https://i.imgur.com/i4bLatn.jpeg"
          style="height: 30em;"
        />
      </div>
      <div
        class="txtContainer col"
      >
        <h2>
          Hosting:
        </h2>
        <ul
          class="leftText scrollText"
          style="border-radius: .5em; color: black;"
        >
          <li>
            Details Magazine Party (Host & Model) - Feb 1997
          </li>
          <li>
            SEMA-AAWI Automotive Convention (Spokesperson) - Nov 1996
          </li>
          <li>
            COMDEX - Aim Tech Corp (Host & Model) - Fall 1995
          </li>
          <li>
            Details Magazine Party (Host & Model) - Jan 1995
          </li>
        </ul>
      </div>
    </div>
    <div
      class="center row"
    >
      <div
        class="col"
      >
        <h2>
          Promotional Materials:
        </h2>
        <ul
          class="leftText scrollText"
          style="border-radius: .5em; color: black;"
        >
          <li>
            Showroom Modeling for PRADA Spring/Summer/Fall 1999 Ready To Wear - Aug 1999
          </li>
          <li>
            (Exclusive Local MAGIC International Trade Show Model - DKNY) - Sept 1998
          </li>
          <li>
            MAGIC International Trade Show (Fall Collection Model - DKNY) - Feb 1998
          </li>
          <li>
            MAGIC International Trade Show (Exclusive Local Model - DKNY) - Aug 1997
          </li>
          <li>
            MAGIC International Trade Show (Model - Venus In Furs) - Aug 1995
          </li>
          <li>
            Action Sports International Trade Show (Model - TAG RAG) - Jan 1995
          </li>
        </ul>
      </div>
      <div
        class="center imgContainer col"
      >
        <img
          alt="Mike White Model"
          src="https://i.imgur.com/V51gsQs.jpeg"
          style="height: 30em;"
        />
      </div>
    </div>
  </div>
  <div
    class="containerBckgroundRed container"
  >
    <p
      style="font-size: larger;"
    >
      _____________________
    </p>
    <img
      alt=" DJ Mike White Logo"
      src="https://i.imgur.com/QLkSBI3.jpeg"
      style="width: 20em;"
    />
    <div
      class="center container"
    >
      <a
        class="socialMedia"
        href="https://soundcloud.com/user-554826787"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-soundcloud fa-2x "
          data-icon="soundcloud"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 640 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M639.8 298.6c-1.3 23.1-11.5 44.8-28.4 60.5s-39.2 24.4-62.3 24.1h-218c-4.8 0-9.4-2-12.8-5.4s-5.3-8-5.3-12.8V130.2c-.2-4 .9-8 3.1-11.4s5.3-6.1 9-7.7c0 0 20.1-13.9 62.3-13.9c25.8 0 51.1 6.9 73.3 20.1c17.3 10.2 32.3 23.8 44.1 40.1s20 34.8 24.2 54.4c7.5-2.1 15.3-3.2 23.1-3.2c11.7-.1 23.3 2.2 34.2 6.7S606.8 226.6 615 235s14.6 18.3 18.9 29.3s6.3 22.6 5.9 34.3zm-354-153.5c.1-1 0-2-.3-2.9s-.8-1.8-1.5-2.6s-1.5-1.3-2.4-1.7s-1.9-.6-2.9-.6s-2 .2-2.9 .6s-1.7 1-2.4 1.7s-1.2 1.6-1.5 2.6s-.4 1.9-.3 2.9c-6 78.9-10.6 152.9 0 231.6c.2 1.7 1 3.3 2.3 4.5s3 1.8 4.7 1.8s3.4-.6 4.7-1.8s2.1-2.8 2.3-4.5c11.3-79.4 6.6-152 0-231.6zm-44 27.3c-.2-1.8-1.1-3.5-2.4-4.7s-3.1-1.9-5-1.9s-3.6 .7-5 1.9s-2.2 2.9-2.4 4.7c-7.9 67.9-7.9 136.5 0 204.4c.3 1.8 1.2 3.4 2.5 4.5s3.1 1.8 4.8 1.8s3.5-.6 4.8-1.8s2.2-2.8 2.5-4.5c8.8-67.8 8.8-136.5 .1-204.4zm-44.3-6.9c-.2-1.8-1-3.4-2.3-4.6s-3-1.8-4.8-1.8s-3.5 .7-4.8 1.8s-2.1 2.8-2.3 4.6c-6.7 72-10.2 139.3 0 211.1c0 1.9 .7 3.7 2.1 5s3.1 2.1 5 2.1s3.7-.7 5-2.1s2.1-3.1 2.1-5c10.5-72.8 7.3-138.2 .1-211.1zm-44 20.6c0-1.9-.8-3.8-2.1-5.2s-3.2-2.1-5.2-2.1s-3.8 .8-5.2 2.1s-2.1 3.2-2.1 5.2c-8.1 63.3-8.1 127.5 0 190.8c.2 1.8 1 3.4 2.4 4.6s3.1 1.9 4.8 1.9s3.5-.7 4.8-1.9s2.2-2.8 2.4-4.6c8.8-63.3 8.9-127.5 .3-190.8zM109 233.7c0-1.9-.8-3.8-2.1-5.1s-3.2-2.1-5.1-2.1s-3.8 .8-5.1 2.1s-2.1 3.2-2.1 5.1c-10.5 49.2-5.5 93.9 .4 143.6c.3 1.6 1.1 3.1 2.3 4.2s2.8 1.7 4.5 1.7s3.2-.6 4.5-1.7s2.1-2.5 2.3-4.2c6.6-50.4 11.6-94.1 .4-143.6zm-44.1-7.5c-.2-1.8-1.1-3.5-2.4-4.8s-3.2-1.9-5-1.9s-3.6 .7-5 1.9s-2.2 2.9-2.4 4.8c-9.3 50.2-6.2 94.4 .3 144.5c.7 7.6 13.6 7.5 14.4 0c7.2-50.9 10.5-93.8 .3-144.5zM20.3 250.8c-.2-1.8-1.1-3.5-2.4-4.8s-3.2-1.9-5-1.9s-3.6 .7-5 1.9s-2.3 2.9-2.4 4.8c-8.5 33.7-5.9 61.6 .6 95.4c.2 1.7 1 3.3 2.3 4.4s2.9 1.8 4.7 1.8s3.4-.6 4.7-1.8s2.1-2.7 2.3-4.4c7.5-34.5 11.2-61.8 .4-95.4z"
            fill="currentColor"
          />
        </svg>
      </a>
      <a
        class="socialMedia"
        href="https://www.instagram.com/djmikewhite_/"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-instagram fa-2x "
          data-icon="instagram"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
            fill="currentColor"
          />
        </svg>
      </a>
      <a
        class="socialMedia"
        href="https://www.facebook.com/mike.white.7739814"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-square-facebook fa-2x "
          data-icon="square-facebook"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M64 32C28.7 32 0 60.7 0 96V416c0 35.3 28.7 64 64 64h98.2V334.2H109.4V256h52.8V222.3c0-87.1 39.4-127.5 125-127.5c16.2 0 44.2 3.2 55.7 6.4V172c-6-.6-16.5-1-29.6-1c-42 0-58.2 15.9-58.2 57.2V256h83.6l-14.4 78.2H255V480H384c35.3 0 64-28.7 64-64V96c0-35.3-28.7-64-64-64H64z"
            fill="currentColor"
          />
        </svg>
      </a>
    </div>
    <p
      style="text-align: right;"
    >
      <a
        href="https://sam-stratton2112.onrender.com"
        style="text-decoration-color: white; color: white; margin-bottom: 2em;"
      >
        © 2024 Samantha Stratton
      </a>
    </p>
  </div>
</DocumentFragment>
`;
